docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -146,6 +146,16 @@ The following provides a good starting point for creating ``etc/jvm.config``:
     # Disable Preventive GC for performance reasons (JDK-8293861)
     -XX:-G1UsePreventiveGC
 
+You must adjust the value for the memory used by Trino, specified with ``-Xmx``
+to the available memory on your nodes. Typically, values representing 70 to 85
+percent of the total available memory is recommended. For example, if all
+workers and the coordinator use nodes with 64GB of RAM, you can use ``-Xmx54G``.
+The rest of the available memory must be sufficient for the operating system and
+other running services. On larger nodes, the percentage value can therefore be
+lower. As an application running on the JVM, Trino does not support using all
+available memory or swap space. Large memory allocation beyond 32GB is
```

Comment:
```suggestion
lower. As an application running on the JVM, Trino does not support using all
available memory or swap space. Large memory allocation beyond 32GB is
```

"All available memory" part is not specific to JVM apps or Trino. Other applications which overprovision memory will also get OOM killed by kernel.

Maybe just keep the statement about disabling swap is highly recommended.

Reply:
Adjusted with some rewording.